### PR TITLE
Correct SPDX-License-Identifier

### PR DESCRIPTION
**Pull request by pshipton**
Issue https://github.com/eclipse-openj9/openj9/issues/17429

### CODE REVIEW

File: JenkinsFile_build_container.groovy [hunk 1]
```diff
@@ -17,9 +17,8 @@
  * [1] https://www.gnu.org/software/classpath/license.html
  * [2] https://openjdk.org/legal/assembly-exception.html
  *
- * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
- * Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
- *******************************************************************************/
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+  *******************************************************************************/
```
Comment (keithc-ca) [inline JenkinsFile_build_container.groovy:21]: Please undo the change to line 21. Similar, unwanted changes were made to `JenkinsFile_build_doc.groovy` and `jenkinsFile_cleanup_staging.groovy`.

Comment (pshipton) [reply JenkinsFile_build_container.groovy]: Updated.

**Review by keithc-ca — APPROVED** [review]
